src/sections/blog/view/singleRecipeView.js

```diff
@@ -195,16 +213,16 @@ export default function SingleRecipeView({
               />
             )}
 
-          {((post.testCasesCode !== undefined && post.testCasesCode !== null) ||
+          {/* {((post.testCasesCode !== undefined && post.testCasesCode !== null) ||
```

Comment:
@franklywatson I have commented out the test cases section for now as all of the recipes are currently using a placeholder test example